tests/testthat/test-sce_to_seurat.R

```diff
@@ -0,0 +1,49 @@
+set.seed(1665)
+sce <- sim_sce(n_cells = 100, n_genes = 200, n_empty = 0)
+colData(sce) <- data.frame("test_column" = sample(0:10, 100, rep= TRUE),
+                          "barcodes" = colnames(sce)) %>%
+  tibble::column_to_rownames("barcodes") %>%
+  DataFrame()
+rowData(sce) <- data.frame("test_row" = sample(0:10, 200, rep = TRUE),
+                           "gene_id" = rownames(sce)) %>%
+  tibble::column_to_rownames("gene_id") %>%
+  DataFrame()
+metadata(sce)$test <- "test"
+alt_sce <- sim_sce(n_cells = 100, n_genes = 10, n_empty = 0)
+# rename the alt_exp so it has the same column names as the base sce
+colnames(alt_sce) <- colnames(sce)
+# merge alt into main
+alt_name = "alt"
+sce <- merge_altexp(sce, alt_sce, alt_name)
+rowData(altExp(sce, alt_name)) <- data.frame("alt_test_row" = sample(0:5, 10, rep = TRUE),
+                                             "alt_id" = rownames(altExp(sce, alt_name))) %>%
+  tibble::column_to_rownames("alt_id") %>%
+  DataFrame()
+
+test_that("Converting SCE to Seurat objects works as expected", {
+
+  if (!requireNamespace("Seurat", quietly = TRUE)) {
+    skip("The Seurat package was not installed. Skipping tests")
+  }
+
+  seurat_object <- sce_to_seurat(sce)
+
+  # check that column names of Seurat object are derived from SCE object
+  # they won't necessarily be equal if some cells with 0 counts were removed
+  expect_true(any(colnames(seurat_object) %in% colnames(sce)))
```

Comment:
The other way around we might not expect a full match, but all of the seurat cells should be in the sce, right?
```suggestion
  expect_true(all(colnames(seurat_object) %in% colnames(sce)))
```